### PR TITLE
mariadb 10.6.5

### PR DESCRIPTION
**Pull request by gfitzp**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
The mroonga patch no longer applies.

**Comment by carlocab** [comment]
Do we no longer need the patch? You'll also need to remove this:

https://github.com/Homebrew/homebrew-core/blob/ae467a4f9cf8342cf2ffa54dff69a8ea47100629/Formula/mariadb.rb#L43-L50

**Comment by gfitzp** [comment]
> Do we no longer need the patch? You'll also need to remove this:
> 
> https://github.com/Homebrew/homebrew-core/blob/ae467a4f9cf8342cf2ffa54dff69a8ea47100629/Formula/mariadb.rb#L43-L50

Ah, sorry, just saw this. I thought the patch was just at the end. I’m not sure if it’s needed or not, it’s the first time I’ve had a bump-formula-pr fail on me, and it sounded like the patch was no longer needed.

File: Formula/mariadb.rb [hunk 1]
```diff
@@ -216,17 +216,3 @@ def plist
 end
 
 __END__
```
Comment (carlocab) [inline Formula/mariadb.rb:218]:
```suggestion
```
No, you need to remove this. Is the patch no longer needed?

Comment (gfitzp) [reply Formula/mariadb.rb:218]: Sorry, I don’t know if it’s needed or not.

**Comment by gfitzp** [comment]
Closing to let someone with better skills attempt a merge 😅

**Pull request closed without merging**